dicoogle/src/main/resources/webapp/js/stores/pluginStore.js

```diff
@@ -2,40 +2,40 @@ import Reflux from "reflux";
 import { Endpoints } from "../constants/endpoints";
 import * as PluginActions from "../actions/pluginActions";
 
-//import {request} from '../handlers/requestHandler';
-
 import dicoogleClient from "dicoogle-client";
 const Dicoogle = dicoogleClient(Endpoints.base);
 
 const PluginStore = Reflux.createStore({
   listenables: PluginActions,
-  init: function() {
+  init: function () {
     this._contents = {};
   },
 
-  onGet: function(type) {
+  onGet: function (type) {
     const self = this;
 
-    Dicoogle.request("GET", ["plugins", type], (error, data) => {
+    Dicoogle.request("GET", ["plugins", type]).end(function (error, data) {
```

Comment:
You can keep using the arrow function syntax, so you can use `this` without having to create an outer `self`.